package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gooddata/react-components",
-  "version": "6.3.0-alpha63",
+  "version": "6.3.0",
```

Comment:
Shouldn't this be `7.0.0`?

Reply:
I don't think so.
'6.3.0' is the state of master right now. 
With the first alpha release of the next branch, it will be rewritten to `7.0.0-alpha1` I think